service_test.go

```diff
@@ -808,7 +808,9 @@ func TestListServicesWithTag(t *testing.T) {
 
 	client := BestTestClient(t, "service/list_with_tag", requests...)
 	// Act
-	response, err := client.ListServicesWithTag(ol.NewTagArgs("app:worker"), nil)
+	tagArgs, err := ol.NewTagArgs("app:worker")
```

Comment:
This should be reverted in line with the changes to the `NewTagArgs()` function signature

Reply:
Will put this in a separate PR